charts/tensorleap/values.yaml

```diff
@@ -49,4 +49,19 @@ keycloak:
       existingClaim: "keycloak-postgresql-data"
   pgchecker:
     image:
-      repository: docker.io/library/busybox # must to add the library before the image name for air-gapped environments
\ No newline at end of file
+      repository: docker.io/library/busybox # must to add the library before the image name for air-gapped environments
+datadog:
+  enabled: true
```

Comment:
We have a flag called `disable-metrics` to disable metrics from the UI to datalog, consider using this flag to disable datadog

Reply:
I do believe that flag is obsolete now, since we don't do UI reporting any more, no?

In any case, wouldn't it be more prudent to have more than one flag? e.g. `enable-datadog-logs`, `enable-datadog-metrics`, `enable-datadog-wui` etc?

Reply:
We still do UI reporting and this flag is currently in use, so you can use it
 It is indeed more prudent to create another flag but in most cases, the user will enable all or disable all so maybe for now I think it is better to stack with just one flag.